zowex/src/proc.rs

```diff
@@ -269,23 +271,23 @@ fn read_pid_for_user() -> Option<sysinfo::Pid> {
  * On Linux:
  *      If you use Stdio::inherit(), you get double output. If use use Stdio::null(), you get no color.
  */
+
 pub fn proc_start_daemon(njs_zowe_path: &str) -> String {
     println!("Starting a background process to increase performance ...");
 
-    if env::consts::OS == "windows" {
-        /* Windows CMD and Powershell terminal windows show escape characters
-         * instead of colors in daemon-mode. A more elegant solution may exist,
-         * but for now we just turn off color in daemon mode on Windows.
-         */
-        env::set_var("FORCE_COLOR", "0");
-    }
-
     let daemon_arg = LAUNCH_DAEMON_OPTION;
-    match Command::new(njs_zowe_path)
-        .arg(daemon_arg)
-        .stdout(Stdio::null())
-        .stderr(Stdio::null())
-        .spawn()
+    let mut cmd = Command::new(njs_zowe_path);
+    
+    // Uses creation flags from https://learn.microsoft.com/en-us/windows/win32/procthread/process-creation-flags
```

Comment:
I am very disappointed that I did not find and use these techniques during the last round of daemon updates. Nicely done!